packages/webpack-cli/lib/utils/resolve-command.js

```diff
@@ -17,6 +17,7 @@ const run = async (name, ...args) => {
             });
         } catch (err) {
             logger.error(`Action Interrupted, use ${cyan('webpack-cli help')} to see possible commands.`);
+            process.exit(2);
```

Comment:
Can we add test

Reply:
I'm not sure how to write a for this

Reply:
unit test inside package, just run `resolve('unknown')`